numba/tests/test_caching.py

```diff
@@ -267,8 +266,11 @@ def run_in_separate_process(self):
             mod.self_test()
             """ % dict(tempdir=self.tempdir, modname=self.modname)
 
+        subp_env = os.environ.copy()
+        subp_env.update(envvars)
         popen = subprocess.Popen([sys.executable, "-c", code],
-                                 stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+                                 stdout=subprocess.PIPE, stderr=subprocess.PIPE,
+                                 env={**os.environ, **envvars})
```

Comment:
```suggestion
                                 env=subp_env)
```
It seems like `subp_env` isn't used, should it go here? Or be deleted?

Reply:
oops, i wanted to switch to `subp_env` because i have to check which dictionary is overriding which in `{**os.environ, **envvars}` spelling.